modules/users/client/controllers/settings/change-password.client.controller.js

```diff
@@ -33,11 +33,13 @@
       // If successful show success message and clear form
       $scope.$broadcast('show-errors-reset', 'vm.passwordForm');
       vm.success = true;
```

Comment:
Removed all vm.success and vm.error and replaced any existing tests for error or success to check for Notification called. All tests passed on my local. Please review carefully :), first time using Jasmine spies.